src/main/java/stsjorbsmod/cards/cull/Frustration.java

```diff
@@ -0,0 +1,61 @@
+package stsjorbsmod.cards.cull;
+
+import com.evacipated.cardcrawl.mod.stslib.StSLib;
+import com.megacrit.cardcrawl.actions.AbstractGameAction;
+import com.megacrit.cardcrawl.actions.common.DamageAllEnemiesAction;
+import com.megacrit.cardcrawl.cards.AbstractCard;
+import com.megacrit.cardcrawl.cards.DamageInfo;
+import com.megacrit.cardcrawl.characters.AbstractPlayer;
+import com.megacrit.cardcrawl.monsters.AbstractMonster;
+import stsjorbsmod.JorbsMod;
+import stsjorbsmod.cards.CustomJorbsModCard;
+import stsjorbsmod.characters.Cull;
+import stsjorbsmod.patches.EntombedField;
+
+public class Frustration extends CustomJorbsModCard {
+    public static final String ID = JorbsMod.makeID(Frustration.class);
+
+    private static final CardRarity RARITY = CardRarity.UNCOMMON;
+    private static final CardTarget TARGET = CardTarget.ALL_ENEMY;
+    private static final CardType TYPE = CardType.ATTACK;
+    public static final CardColor COLOR = Cull.Enums.CULL_CARD_COLOR;
+
+    private static final int COST = 1;
+    private static final int DAMAGE = 15;
+    private static final int UPGRADE_DAMAGE = 3;
+    private static final int DAMAGE_ON_RETAIN = 2;
+    private static final int UPGRADE_DAMAGE_ON_RETAIN = 1;
+
+
+    public Frustration() {
+        super(ID, COST, TYPE, COLOR, RARITY, TARGET);
+        this.selfRetain = true;
+        this.isMultiDamage = true;
+        this.baseDamage = DAMAGE;
+        this.baseMagicNumber = DAMAGE_ON_RETAIN;
+        this.exhaust = true;
+        this.exert = true;
+    }
+
+
+    @Override
+    public void onRetained() {
+        this.addToBot(new DamageAllEnemiesAction(null, DamageInfo.createDamageMatrix(magicNumber, true), DamageInfo.DamageType.THORNS, AbstractGameAction.AttackEffect.FIRE));
+    }
+
+    @Override
+    public void use(AbstractPlayer p, AbstractMonster abstractMonster) {
+        this.addToBot(new DamageAllEnemiesAction(p, this.multiDamage, this.damageTypeForTurn, AbstractGameAction.AttackEffect.SLASH_DIAGONAL));
+
+        AbstractCard masterCard = StSLib.getMasterDeckEquivalent(this);
```

Comment:
I would probably lean towards trying to treat this more like how `this.exhaust` works, where if `this.exert` is true, the card exhausts and sets its master deck equivalent to "exerted" automatically when played, rather than requiring each individual card to set all of that explicitly.

I think it'd be okay to do a similar compromise option to what `EphemeralField` does, where setting `ExertField` to true on a card implicitly also sets `card.exhaust` to true.

Reply:
Agreed. Where do I actually patch in the stuff that causes the card's ExertedField to get set? Surely it's not just a Postfix on AbstractCard::Use, right? I'm trying to find the spot where a card with this.exhaust triggers a move to the exhaust pile, but I can't seem to find it.

Reply:
Ah, found it in UseCardAction (makes sense, I just had never seen this being called). I'll see if I can figure out how to patch it. But that's not my forte, so any help is much appreciated :)

Reply:
Alright, I think I figured it out. Hope it's not too crude a solution.

Reply:
LGTM!